CHANGES

```diff
@@ -22,9 +22,28 @@ Incompatible changes
 Deprecated
 ----------
 
+* #10028: jQuery and underscore.js will no longer be automatically injected into
+  themes from Sphinx 6.0. If your theme's JavaScript uses the ``jQuery``,
+  ``$``, or ``$u`` global objects, you need to update your theme. To add jQuery
+  and underscore.js back to your theme, add the following code to your
+  ``layout.html``.
+
+  .. code-block:: html+jinja
+
+     {%- block scripts %}
+         <script src="{{ pathto('_static/jquery.js', resource=True) }}"></script>
+         <script src="{{ pathto('_static/underscore.js', resource=True) }}"></script>
+         {{ super() }}
+     {%- endblock %}
```

Comment:
It's worth noting that this workaround will not work, starting with Sphinx 6.0.

Reply:
Is this as the source files themselves will be removed from the repo? If so good point -- I should add a note that downsteam authors should add the sources to their `static` directories. (If not, not sure I understand you, sorry!)

Reply:
updated note to say that downstream must copy the sources from this project